src/PrestaShopBundle/Resources/config/services/core/form/form_data_provider.yml

```diff
@@ -46,7 +46,7 @@ services:
       - '@prestashop.core.query_bus'
       - '@=service("prestashop.adapter.shop.context").getContextListShopID()'
       - '@=service("prestashop.adapter.legacy.context").getContext().shop.getCategory()'
-      - '@prestashop.adapter.group.provider.default_groups_provider'
+      - '@prestashop.adapter.group.group_data_provider'
```

Comment:
You should maybe change `prestashop.core.form.identifiable_object.data_provider.customer_form_data_provider` and deprecated `prestashop.adapter.group.provider.default_groups_provider`.

ping @matks

Reply:
I don't understand, default_groups_provider is still useful for customer_form_data_provider

Reply:
Can you please explain what you have in mind @PierreRambaud ?

Reply:
I mean, because we are waiting for the same behavior in customer and category when reaching groups, no?

Reply:
No, It make no sense to me. Is there an issue on this ?

Reply:
@PierreRambaud at customer creation time, it make sense to me to suggest only the customer group or no group at all.